source/Nevermore/IRelationalStore.cs

```diff
@@ -11,5 +12,7 @@ public interface IRelationalStore
         IRelationalTransaction BeginTransaction(RetriableOperation retriableOperation = RetriableOperation.Delete | RetriableOperation.Select, string name = null);
         IRelationalTransaction BeginTransaction(IsolationLevel isolationLevel, RetriableOperation retriableOperation = RetriableOperation.Delete | RetriableOperation.Select, string name = null);
         void WriteCurrentTransactions(StringBuilder sb);
+        DocumentMap GetMappingFor<T>();
+        DocumentMap GetMappingFor(Type type);
```

Comment:
This one could be an extension method to keep the interface slim ?

Reply:
Not without making the mapping object public. Also it'd be a PITA to mock

Reply:
I actually meant the `DocumentMap GetMappingFor<T>();`

Reply:
They forward to different methods (as confusing as that is :/)

Reply:
Should this be a TryGetMappingFor ?